### PR TITLE
Update React Native to 0.55.X

### PR DESCRIPTION
**Pull request by saghul**
https://github.com/jitsi/lib-jitsi-meet/pull/747

### CODE REVIEW

**Comment by saghul** [comment]
Jenkins, please test this please.

File: react/features/base/jwt/components/CalleeInfo.js [hunk 1]
```diff
@@ -75,9 +75,9 @@ class CalleeInfo extends Component<Props, State> {
 
     _onLargeVideoAvatarVisible: Function;
 
-    _playAudioInterval: ?number;
+    _playAudioInterval: *;
```
Comment (paweldomas) [inline react/features/base/jwt/components/CalleeInfo.js]: `?InternvalID` works for me here (declared in flow core)

File: react/features/base/jwt/components/CalleeInfo.js [hunk 2]
```diff
@@ -75,9 +75,9 @@ class CalleeInfo extends Component<Props, State> {
...
 
-    _ringingTimeout: ?number
+    _ringingTimeout: *;
```
Comment (paweldomas) [inline react/features/base/jwt/components/CalleeInfo.js]: `?TimeoutID` declared by flow core works for me here

File: react/features/calendar-sync/components/ConferenceNotification.native.js [hunk 3]
```diff
@@ -56,14 +56,14 @@ type State = {
  * screen when another meeting is about to start.
  */
 class ConferenceNotification extends Component<Props, State> {
-    updateIntervalId: number;
+    updateIntervalId: *;
```
Comment (paweldomas) [inline react/features/calendar-sync/components/ConferenceNotification.native.js]: `IntervalID` seems to be working here

**Comment by saghul** [comment]
@lyubomir Do you want me to hold this or can I merge it already?

**Comment by saghul** [comment]
Rebased since there was a new conflict caused by status-indicators landing.

**Pull request merged**